src/components/LayerList.tsx

```diff
@@ -310,6 +317,9 @@ class LayerListContainer extends React.Component<LayerListContainerProps, LayerL
   }
 }
 
+// eslint-disable-next-line react-refresh/only-export-components
```

Comment:
Would be great if you could find a way to fix these lint issues, I didn't find a better way to do it beside ignore the lint issue, but it's not a great solution.